option.go

```diff
@@ -71,3 +71,14 @@ type nopReplayDetector struct{}
 func (s *nopReplayDetector) Check(uint64) (func() bool, bool) {
 	return func() bool { return true }, true
 }
+
+// MasterKeyIndicator sets MKI for RTP and RTCP.
```

Comment:
It would be better to mention that `mki` passed to the option is a MKI for the initial master key and any MKIs added later must have same length.

Reply:
Updated comment.